src/lib/y2storage/planned_volume.rb

```diff
@@ -188,11 +188,12 @@ def create_filesystem(partition)
     #
     # @param filesystem [::Storage::Filesystem]
     def create_subvolumes(filesystem)
-      return unless filesystem.btrfs?
+      return unless filesystem.type == ::Storage::FsType_BTRFS
```

Comment:
Yes, but this doesn't use the new API at all yet. We'll get to that at some later time.